TCMSApp/src/client/app/tests/tests-create.html

```diff
@@ -22,9 +22,7 @@ <h1 class="text-center">Create New Test Case</h1>
               <div class="form-group">
                 <label for="caseSuite">Suite:</label>
                 <select class="form-control" id="caseSuite" name="caseSuite">
-                  <option>Suite 1</option>
-                  <option>Suite 2</option>
-                  <option>Suite 3</option>
+                  <option>{{ ::vmCase.currentSuite.suiteName }}</option>
                   <option>None</option>
```

Comment:
option None? :) why?

we shoul deside if we can change the suite for test case in creation phase or not. 
But there should not be tests with no link to suite